src/highlight/__tests__/HighlightContainer-test.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { IntlShape } from 'react-intl';
+import { mount, ReactWrapper } from 'enzyme';
+import HighlightAnnotations from '../HighlightAnnotations';
+import HighlightContainer, { Props } from '../HighlightContainer';
+import { createStore } from '../../store';
+
+jest.mock('../../common/withProviders');
+jest.mock('../HighlightAnnotations');
+
+describe('HighlightContainer', () => {
```

Comment:
Should we follow the path format here? I'd also be game for nuking the path entirely and just using the component name. Maintaining the path here seems tedious.

Reply:
Will nuke the path entirely in the followup pr.